source/tools/glkernel-cmd/main.cpp

```diff
@@ -0,0 +1,58 @@
+#include "KernelGeneration.h"
+#include "KernelToJson.h"
+
+#include <iostream>
+#include <fstream>
+
+#include <cppassist/cmdline/ArgumentParser.h>
+
+#include <cppexpose/variant/Variant.h>
+#include <cppexpose/json/JSON.h>
+
+int main(int argc, char* argv[])
+{
+    cppassist::ArgumentParser argParser;
+    argParser.parse(argc, argv);
```

Comment:
You should add the output of a help text if no parameter or the `--help` parameter is passed. `ArgumentParser::print()` may be a good starting point for this.